### ISSUE TITLE
Missing responseparser.php

### ISSUE THREAD

**Issue by micomator** [rotorcowboy/Phapper#3]

**Comment by codeandbattle** [comment]
Yup. And removing/commenting the require_once for it just produces another error. I cannot find that file in older commits either.

**Comment by codeandbattle** [comment]
https://github.com/rotorcowboy/Phapper/commit/5e03e18bce3ca1be8448632ad736cdde8ab1eacf

This appears to be the commit where they added responseparser.php to phapper.php - but even in this commit the file is not there.

**Comment by Voral** [comment]
+1

**Comment by jorgegil96** [comment]
+1

**Comment by jorgegil96** [comment]
I got it to work by removing that line and didn't get any error besides the ones for not having a config.php file

**Comment by kishan-v** [comment]
Hope someone could help me on this:

Relatively new to OO Api's. I have followed the get started so far, and I have the above issue however if i remove

`   require_once(__DIR__.'/inc/responseparser.php');
`
 i get the following errors:

```
PHP Fatal error:  Uncaught Error: Class 'PhapperConfig' not found in D:\Documents\Reddit\Phapper-master\src\phapper.php:28
Stack trace:
#0 D:\Documents\Reddit\Phapper-master\fileThatDoesGetMe.php(4): Phapper->__construct()
#1 {main}
  thrown in D:\Documents\Reddit\Phapper-master\src\phapper.php on line 28

```

How did you guys ( @jorgegil96 ) manage to fix your errors? Like i said i tried removing the require once line as you said.
Let me know if you need screenshots or anything else
Thank You
Kish

**Comment by jorgegil96** [comment]
You need to create your own version of the PhapperConfig. There's a sample file config.sample.php https://github.com/rotorcowboy/Phapper/blob/master/config.sample.php

**Comment by kishan-v** [comment]
Thanks a lot @jorgegil96 that works now, however i have a new issue now:

https://github.com/rotorcowboy/Phapper/issues/5
